Confus/Confus/Flag.cpp

```diff
@@ -2,16 +2,15 @@
 #include <IrrAssimp/IrrAssimp.h>
 #include <iostream>
 
-#include "Flag.h"
+#include <RakNet/BitStream.h>
 #include "Player.h"
 #include "ClientTeamScore.h"
 
-#include "../ConfusShared/Physics/PhysicsWorld.h"
-#include "../ConfusShared/Physics/BoxCollider.h"
+#include "Flag.h"
 
 namespace Confus 
 {
-	Flag::Flag(irr::IrrlichtDevice* a_Device, ETeamIdentifier a_TeamIdentifier, Physics::PhysicsWorld& a_PhysicsWorld) : 
+	Flag::Flag(irr::IrrlichtDevice* a_Device, ETeamIdentifier a_TeamIdentifier) : 
```

Comment:
This seems like some major merge conflict? It seems to remove the entire physics functionality that got in recently